bootstrap/fabric/build.gradle.kts

```diff
@@ -92,6 +94,15 @@ tasks {
         archiveClassifier.set("")
         archiveVersion.set("")
```

Comment:
Can you flip this to match the order in the `remapModrinthJar`

The order in `remapModrinthJar` is good because it's the same order as the syntax that shadow generates
`[archiveBaseName]-[archiveAppendix]-[archiveVersion]-[archiveClassifier].[archiveExtension]`